mappings/net/minecraft/client/world/ClientChunkManager.mapping

```diff
@@ -24,6 +24,7 @@ CLASS dgo net/minecraft/client/world/ClientChunkManager
 	FIELD g playerChunkZ I
 	FIELD h world Ldgt;
 	METHOD a updateChunkList ()V
+	METHOD a updateLoadDistance (I)V
```

Comment:
Why not `updateChunkLoadDistance`, to match the packet name?

Reply:
It is actually updating the chunk load distance to `arg1 + 2`, and I figured that this is already a chunk manager, so I will just say load distance.